### ISSUE TITLE
Is there way to override app notifications settings for android ?

### ISSUE THREAD

**Issue by webcodepl** [dpa99c/cordova-plugin-firebasex#329]
I have app using this plugin. Default in android notifications settings for my app i have only enabled parameters 'show notifications' and 'allow vibration'. 

![fcm_settings](https://user-images.githubusercontent.com/11521576/75988578-196ea680-5ef2-11ea-8154-cfe1e4b22010.jpg)

Notifications send from firebase console page only show in status bar (with no sound and other settings from defined channel).
If i want my app to use defined channel settings (sound, badges, led light), i have to manually enable the other parameters first in settings>notifications>myapp

I realized that apps like Messenger Lite or WhatsApp Messenger have theses parameters enabled immediately after installation. 

![messenger_settings](https://user-images.githubusercontent.com/11521576/75988595-1ffd1e00-5ef2-11ea-8252-eac82933e118.jpg)

How can my app have theses parameters enabled by default ?

**Comment by dpa99c** [comment]
For Android 8+ you need to define a [notification channel](https://github.com/dpa99c/cordova-plugin-firebasex#android-notification-channels) using [createChannel()](https://github.com/dpa99c/cordova-plugin-firebasex#createchannel) then specify that channel when sending the FCM notification as demonstrated in this [test message](https://github.com/dpa99c/cordova-plugin-firebasex-test/blob/master/messages/android_notification_custom_channel.json#L10).

You can build and run the [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) and send the test messages using its built-in FCM client to try this out.

**Comment by webcodepl** [comment]
I use code below and settings of the created channel are taken only when i first manually enable parameters in settings>notifications>myapp. If i dont, notifications behave as send without channel_id property.

```javascript
var customChannel  = {
	id: "fcmtest",
	name: "FCMTest",
	sound: "ringtone",
	vibration: [1000, 1000, 1000, 1000, 1000],
	light: true,
	lightColor: "0xFF0000FF",
	importance: 4,
	badge: true,
	visibility: 1
};

FirebasePlugin.createChannel(customChannel,
	function() {
		console.log("Created custom channel: " + customChannel.id);
		FirebasePlugin.listChannels(
			function(channels) {
				if(typeof channels == "undefined") return;
				for(var i=0;i<channels.length;i++) {
					console.log("Channel id=" + channels[i].id + "; name=" + channels[i].name);
				}
			},
			function(error) {
				console.error('List channels error');
			}
		);
	},
	function(error) {
		console.error("Create channel error", error);
	}
);

FirebasePlugin.setDefaultChannel(customChannel,
  function() {
	  console.log('Default channel set');
  },
  function(error) {
	 console.log('Set default channel error: ' + error);
  }
);

```

**Comment by dpa99c** [comment]
Please build and run the [example project](https://github.com/dpa99c/cordova-plugin-firebasex-test) and use it as a reference, sending the test messages using its built-in FCM client.

**Comment by webcodepl** [comment]
I built example project and sent notifications/data using its FCM client. Still the same. Notifications show only in status bar and on console. I have to manually enable parameters in settings>notifications>example (allow sound, allow led light, icon badges) to include settings from incoming notifications. 
One more thing. I tested this on Xiaomi Redmi 4a (android 7) and 6a (android 9). When i launched this example on galaxy s9, everything works immediately. I didn't have to change any notifications settings. Maybe this is xiaomi/miui issue.
